extract-wos-article.py

```diff
@@ -23,7 +30,54 @@
     "Current Publisher": "publisher",
 }
 
-YEAR_REGEX = re.compile(".*([0-9]{4})")
+REGEX = {
+    "year": re.compile(".*([0-9]{4})"),
+    "wos_id_number": re.compile("([0-9]{15})"),
+}
+CITING_SUMMARY_URL_PREFIX = "https://www.webofscience.com/wos/woscc/citing-summary/"
+
+
+async def extract_citing_summary(page: Page, article_id: str):
+    url = CITING_SUMMARY_URL_PREFIX + article_id
```

Comment:
I don't really like the `+` here but then again, it doesn't matter all that much :slightly_smiling_face:

Reply:
@mrshu why don't you like it? i've fixed it with f-string, 'cause it's more pythonish imho, but I'm curios what do you think.

Reply:
@JMatej It's no more than a matter of preference -- that `+` there was a bit semantically off with regards to what is happening (appending the `article_id` to the `CITING_SUMMARY_URL_PREFIX`).

That said, no need to go over it again :slightly_smiling_face: